pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
Of course if `git` version render people unable to play multiplayer it would be nice to have release instead. For other example, we try to keep both source builds and binary packages of `zandronum` (a Doom engine) just because developers marked binary release differently than the source package, so they refuse to connect to each other (and majority of servers from my experience are using binary release).